apps/src/codegen.js

```diff
@@ -300,8 +239,12 @@ export function marshalNativeToInterpreter(interpreter, nativeVar, nativeParentO
   if (typeof maxDepth === "undefined") {
     maxDepth = Infinity; // default to infinite levels of depth
   }
-  if (shouldCustomMarshalObject(nativeVar, nativeParentObj)) {
-    return createCustomMarshalObject(interpreter, nativeVar, nativeParentObj);
+  // TODO (pcardune): remove circular dependency
```

Comment:
This TODO is still around after your last batch of changes.  I assume you'd like to address it in the future?